pyproject.toml

```diff
@@ -7,3 +7,10 @@ exclude = [
   "docs",
   "build",
 ]
+
+[tool.isort]
+profile = "black"
+import_heading_stdlib="Standard Library"
+import_heading_firstparty="Sematic"
+known_third_party = ["flask", "psycopg2", "matplotlib", "cloudpickle"]
```

Comment:
It *may* have been some weird issue based on the fact that this was the first time we ran on formerly unsorted things, but it mis-placed these in a few files on the first execution. Don't really know why, we can look into it if it starts to misplace other things.